website/blog/2021-06-24-introduce-superbench.md

```diff
@@ -8,4 +8,113 @@ author_url: https://github.com/TobeyQin
+
+For more information, please view [confiuration](https://microsoft.github.io/superbenchmark/docs/getting-started/configuration)
+
+### Modular and Extensible Framework
+
+1. Executor Framework
+
+In order to facilitate the benchmarking and validation of large-scale clusters, we designed and implemented a modular and extensible framework. 
+SuperBench framework includes a runner as control node, as well as multiple executors as worker nodes. 
+A runner received commands from CLI and distribute to all nodes (worker nodes) in the cluster and collect results. 
+Worker nodes will execute the specified benchmark tasks.
+
+![SuperBench Executor Workflow](../../docs/assets/executor_workflow.png)
+
+2. Benchmark Framework
+
+SuperBench supports micro-benchmark for primitive computation and communication benchmarking,
+and model-benchmark to measure domain-aware end-to-end deep learning workloads.
+SuperBench implemented an abstract BenchmarkBase to provide common fuction, while other benchmarks are plugged-in as components.
+It also provides a unified entrance and result format for all benchmarks.
+Developers can easily add new benchmarks.
+
+![SuperBench Benchmark Package](../../docs/assets/benchmark_package.png)
+
+### Conprehensive and Strandardized Benchmarks
+
+SuperBench supports a set of benchmarks listed as below.
+
+* Micro-Benchmarks
+  * Computation benchmarks
+    * GEMM Flops
+    * Kernel Launch Time
+    * Operator Performance
+    * Memory
+  * Communication benchmarks
+    * Device P2P
+    * RDMA
+    * NCCL
+  * Computation-Communication Benchmarks
+  * Storage Benchmarks
+
+* Model-Benchmarks
+  * CNN models
+  * LSTM models
+  * BERT models
+  * GPT-2 models
+
+For the details of each benchmark, please view [micro-benchmarks](https://github.com/microsoft/superbenchmark/blob/main/docs/benchmarks/micro-benchmarks.md) 
+and [model-benchmarks](https://github.com/microsoft/superbenchmark/blob/main/docs/benchmarks/model-benchmarks.md).
+
+
+## What's next?
+
+We want to exend superbench capability to auto-diagnosis and simulation, to build a benchmarking eco-system.
+The following figure shows the whole picture.
+![SuperBench Capabilities and Extension](../../docs/assets/architecture.png)
+
+With SuperBench and its extensions, we can support:
+
+* AI infrastructure validation and diagnosis
+  * Distributed validation tools to validate hundreds or thousands of servers automatically
+  * Consider both raw hardware and E2E model performance with ML workload patterns
+  * Build a contract to identify hardware issues
+  * Provide infrastructural-oriented criteria as Performance/Quality Gates for hardware and system release
+  * Provide detailed performance report and advanced analysis tool
+* AI workload benchmarking and profiling
+  * Provide comprehensive performance comparison between different existing hardware
+  * Provide insights for hardware and software co-design
```

Comment:
* Quick and Trustable Distributed Validation 
  * Distributed validation tools to validate hundreds or thousands of servers automatically
  * Provide minute-level fast validation and guarantee high repeatability for each benchmarks   
  * Provide baseline for different systems as Performance/Quality Gates for hardware and system release
* Detailed Auto Diagnosis
  * Provide comprehensive diagnosis benchmarks to analyze the detailed issued on defective node
  * Provide detailed performance report and advanced analysis tool

Reply:
changed